modules/motions/ui/MotionFormStartNew/Parts/StartSDVTNodeOperatorsAdd.tsx

```diff
@@ -185,11 +199,18 @@ export const formParts = ({
                     name={`${fieldNames.nodeOperators}.${i}.rewardAddress`}
                     rules={{
                       required: 'Field is required',
-                      validate: value => {
-                        if (!utils.isAddress(value))
+                      validate: async value => {
+                        if (!utils.isAddress(value)) {
                           return 'Address is not valid'
-                        if (value === constants.AddressZero)
+                        }
+                        if (value === constants.AddressZero) {
                           return 'Should not be zero address'
+                        }
+                        const isLidoRewardAddress =
```

Comment:
This check ensures that input address is not equal to `LidoLocator` contract address. As [this doc](https://hackmd.io/@bkovtun/rkiXmDE02#AddNodeOperators) says: 
> Reward addresses of newly added node operators MUST NOT contain the address of the stETH token

We should compare the address with stETH token address instead.